source/get-started/bare-metal-install-server.rst

```diff
@@ -70,8 +70,28 @@ Follow these steps to install |CL| on the target system:
 
       Figure 1: Clear Linux OS Installer boot menu
 
+#. Follow `Verify integrity of installer media (optional)`_.
+
 #. With :guilabel:`Clear Linux OS` highlighted, select :kbd:`Enter`.
 
+
+Verify integrity of installer media (optional)
+**********************************************
+
+Use :guilabel:`Verify ISO Integrity` to verify the checksum of 
+the image burned to the installer media is the same one used at boot time 
+and is not corrupted (see Figure 1). For every ISO generated, the 
+:guilabel:`clr-installer` implants checksums, which are verified during 
+early boot stage as part of :command:`initrd`. 
+ 
+#. Select :guilabel:`Verify ISO Integrity`. The media will be validated. 
+
+#. If the check passes, it will boot into the live image. 
+
+#. If the check fails, a failure message appears. 
+   
+   #. Follow `Preliminary Steps`_. 
```

Comment:
Isnt this supposed to point to a section which says something like trying reburning your image to the media?? Are you yet to write that?

Reply:
`Preliminary steps` is a hard link. Basically, if the image fails the checksum, the user has to return to the Preliminary steps and download a new image.  In my latest commit, I reworded that sentence as:  

```
Restart the process at `Preliminary Steps`_. 
```